Applications/Utils/SWMMConverter/SWMMInterface.cpp

```diff
@@ -0,0 +1,36 @@
+
+
+#include <string>
+#include <iostream>
+#include "ThirdParty/SWMMInterface/swmm5_iface.h"
+
+int main(int argc, char *argv[])
+{
+   // Open outfile as a SWMM output file
+    std::string outfile = argv[1];
```

Comment:
If no command line argument is given the tool will probably crash without a useful error message.